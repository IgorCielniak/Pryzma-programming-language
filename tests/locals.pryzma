/func{loc a = 9; @func2(&a)}

/func2{b = *args[0]; print b}

@func

/f{ print b, "\n" }

@f

print a

