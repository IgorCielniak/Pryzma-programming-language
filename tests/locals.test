python Pryzma.py tests/locals.pryzma
