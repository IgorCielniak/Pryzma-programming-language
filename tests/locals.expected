99
Error near line 11: Variable 'a' not found in current scope.
None